src/components/console/RequestList/Item.js

```diff
@@ -0,0 +1,40 @@
+import { useStyles } from './Item.style'
+import { useConsoleContext } from 'src/context/ConsoleContext'
+import React from 'react'
+import clsx from 'clsx'
+import { useScrollWithDropdown } from './useScrollWithDropdown'
+import dots from '../../../icons/dots.svg'
+import green from '../../../icons/EllipseGreen.svg'
+import red from '../../../icons/EllipseRed.svg'
+
+function Item({name, success, customref, onDelete}) {
+    const {deleteRequest, copyRequest, doRequest} = useConsoleContext()
+    const styles = useStyles()
+    const [buttonRef, ulRef] = useScrollWithDropdown();
+
+    return (
+        <div  className={styles.rel} >
+            <div className={styles.imgWrap}>
+                {success ? <img src={green} alt="green" />: <img src={red} alt="red" />}
+            </div>
+            <span>{name}</span>
+            <div ref={buttonRef} className={styles.dotsWrap}>
+                <img src={dots} alt="dots" />
+                <ul ref={ulRef} 
+                    className={styles.dropdown}>
+                    <div className={styles.buttonWrapper} >
+                        <button onClick={()=>doRequest(name)} className={clsx(styles.dropdownButton,styles.blueButton)}>Выполнить</button>
+                    </div>  
+                    <div className={styles.buttonWrapper} >
+                        <button onClick={()=>copyRequest(name)} className={clsx(styles.dropdownButton,styles.blueButton)}>Скопировать</button>
+                    </div> 
+                    <div className={styles.buttonWrapper} >
+                        <button onClick={()=>deleteRequest(name)} className={clsx(styles.dropdownButton,styles.redButton)}>Удалить</button>
+                    </div>   
```

Comment:
As you see, you use almost the same divs with same styles, the only thing  different is function that you call. Leave it like that but for a future, you can simply create a variable that gonna return the jsx with only one div and you conditionally pass the functions there.

Reply:
this comment is done

Reply:
LGTM, only put naming into separate file!!!

Reply:
This comment is fixed

Reply:
LGTM